common/cmake/devices.cmake

```diff
@@ -884,6 +894,8 @@ function(ADD_FPGA_TARGET)
   set(OUT_ROUTE ${OUT_LOCAL}/${TOP}.route)
 
   set(VPR_DEPS "")
+  list(APPEND VPR_DEPS ${OUT_EBLIF})
```

Comment:
Use "append_file_dependencies(${OUT_EBLIF_REL})" over this

Reply:
Done